### DISCUSSION TITLE
z ve o sorusu

### DISCUSSION THREAD

**Discussion by Ersan0102** [sonsuzus/sonsuzus#426] [Soru Cevap]
**Problem:**
Şunlardan oluşan bir kelime girmeniz gerekmektedir:
 x ve y
 bu sırasıyla Z ve O sayısını belirtir. Giriş kelimesi, eğer 2x = y ise, hayvanat bahçesi kelimesine benzer kabul edilir.

Girilen kelimenin hayvanat bahçesi kelimesine benzer olup olmadığını belirleyin.

Örneğin, zzoooo ve zzzooooo gibi kelimeler zoo kelimesine benzer ancak zzooo ve zzzooooo gibi kelimeler değildir.

**Giriş formatı**

İlk satır: Birkaç Z ile başlayan ve birkaç O ile devam eden bir kelime.
Not: Bu kelimenin maksimum uzunluğu 20 olmalıdır.

**Çıkış biçimi**

Eğer giriş kelimesi hayvanat bahçesi dizisi olarak kabul edilebilirse evet yazdır , aksi takdirde Hayır yazdırın.
**Benzer giriş: **
zzzoooooo

**Benzer çıkış.:**
evet
çünkü o sayısı z sayısının iki katıdır

**Comment by Ersan0102** [comment]
Linki: https://www.hackerearth.com/practice/basic-programming/input-output/basics-of-input-output/practice-problems/algorithm/is-zoo-f6f309e7/

**Comment by Erva-Naz** [comment]
#include <iostream>
#include<bits/stdc++.h>
using namespace std;

int main() {
	
string str;

int z_deg=0, o_deg=0;

cin>>str;

for(int i=0;i<str.length();i++) {

	if(str[i] =='z'){
		z_deg++;
	}

	else{
		o_deg++;
	}
}
if(2*z_deg == o_deg) {
	cout<< "Yes"<<endl;
}
else {
	cout << "No"<<endl;
}

}
